cmd/buildtools/helm.go

```diff
@@ -188,3 +195,59 @@ func (h *Helm) Push(path, dst string) error {
 	}
 	return up.UploadTo(path, dst)
 }
+
+func (h *Helm) Render(chartName string, chartPath string, vals map[string]interface{}, namespace string) ([]string, error) {
+
+	const k8sVersionMajor, k8sVersionMinor = "1", "26"
```

Comment:
Maybe we should use 1.28 as this is the oldest ec version we have. I recognise that I don't fully understand the impact of this, I assume it will render only types that are known to 1.28 (maybe do some conversion?). Is that it ?